engine/src/api/workflow/agents/planner/types.py

```
@@ -1,30 +1,50 @@
-from typing import Dict, Any, List
-from pydantic import Field, BaseModel
+"""Pydantic models and types for planner agent."""
 
+from typing import Dict, Any, List, Optional
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)